Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use an official Python runtime based on Debian 10 "buster" as a parent image.
-FROM python:3.8.1-slim-buster
+FROM python:3.8.1-slim-buster as app
```

Comment:
for my info: what does this do? (A quick google is not informative)

Reply:
This aliases the image name to `app`, so in `docker-compose.test.yml:13` we can reference that instead of the full image name.